Podcast/EpisodeCard.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  EpisodeCard.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 2/22/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import SwiftyJSON
+
+class EpisodeCard: Card {
+    
+    var episodeID: Int
+    var episodeTitle: String = ""
+    var dateCreated: Date!
+    var time: Double!
+    var descriptionText: String!
+    var smallArtworkImageURL: URL?
+    var smallArtworkImage: UIImage?
+    var nRecommended: Int //number total out of user base that has recommended episode
+    var tags: [String]
+    var seriesTitle: String!
+    var isBookmarked: Bool!
+    var isRecommended: Bool!
+    var isPlaying: Bool! = false
+    
+    init(episodeID: Int, episodeTitle: String = "", dateCreated: Date = Date(), descriptionText: String = "Not avaliable", smallArtworkImage: UIImage = #imageLiteral(resourceName: "filler_image"), time: Double = 0.0, nRecommended: Int = 0, tags: [String] = [], seriesTitle: String = "", isBookmarked: Bool = false, isRecommended: Bool = false) {
+        self.episodeID = episodeID
+        self.episodeTitle = episodeTitle
+        self.dateCreated = dateCreated
+        self.descriptionText = descriptionText
+        self.smallArtworkImage = smallArtworkImage
```

Comment:
why do you pass in the image instead of image url? should be the image url, which you can leave for now, but we will need to load asynchronously using haneke later. Change this to imageUrl